Assets/Scripts/Visuals/UiHierarchyVisuals.cs

```diff
@@ -113,8 +113,15 @@ private void UpdateVisuals()
             }
             else
             {
-                MakeHierarchyItemsRecursive(scene, 0, scene.EntitiesInSceneRoot, mainPanelData);
-
+                if (scene.EntitiesInSceneRoot.Count() == 0)
+                {
+                    mainPanelData.AddTitle("No Entities");
+                }
+                else
+                {
+                    MakeHierarchyItemsRecursive(scene, 0, scene.EntitiesInSceneRoot, mainPanelData);
+                }
+                
                 mainPanelData.AddSpacer(300, clickPosition =>
```

Comment:
Weird behavior, even if I move this AddSpacer next to line 122, the right click behavior is still there.

I don't know what this AddSpacer does.

Reply:
![grafik](https://user-images.githubusercontent.com/11379989/217663583-f511bf32-8bce-4bb4-be53-fce1f0488146.png)
This is because the Viewport, that is the parent of the hierarchy, also has a right-click handler. 
I think, being able to right-click here is the behavior we want, so that is fine.